### ISSUE TITLE
How to set the base_learning_rate in finetune autoencoderKL?

### ISSUE THREAD

**Issue by haooxia** [CompVis/latent-diffusion#310]
How to set the base_learning_rate in finetune autoencoder?
Should I modify the default `base_learning_rate: 1.6e-7`

It can be seen that the learning rate changes with num_gpus, batchsize and base_learning_rate:
![image](https://github.com/CompVis/latent-diffusion/assets/62507136/e4e0be1a-af05-4f10-9458-eebbb9dbd775)

**Comment by OwalnutO** [comment]
Set `--scale_lr=False`, then the learning rate will not be modified.

**Comment by haooxia** [comment]
> Set `--scale_lr=False`, then the learning rate will not be modified.

Thanks for the reply, the `base_learning_rate` given by the repository is [4.5e-6](https://github.com/CompVis/latent-diffusion/blob/a506df5756472e2ebaf9078affdde2c4f1502cd4/configs/autoencoder/autoencoder_kl_64x64x3.yaml#L2C23-L2C29), so what value should I shrink to when finetune on my own dataset?
